inc/commondbtm.class.php

```diff
@@ -3099,20 +3101,6 @@ function isPrivate() {
       return false;
    }
 
-   /**
```

Comment:
i disagree with this change.

At least, the method shool be kept in `CommonDBTM` and return `false`, (the trail will override this method).
Because, calls (like https://github.com/glpi-project/glpi/blob/9.4.4/inc/search.class.php#L1579) will fail

Reply:
Tee default function is now on top of parent class -> CommonGLPI
Because, if we do this in CommonDBTM, we need to add ```use MapGeolocation;``` on each sub class of CommonDBTM.

Reply:
I do not understand why this call would fail. Moving method into a trait used by a class should not affect calls to this method.

Reply:
as discussed,

maybelocated function must remain on commonDBTM 
and trait must be used to display the map.